docs/src/main/sphinx/connector/druid.rst

```diff
@@ -57,6 +57,50 @@ name from the properties file.
 Type mapping
 ------------
 
+Because Trino and Druid each support types that the other does not, this
+connector modifies some types when reading or writing data. Data types may not
+map the same way in both directions between Trino and the data source. Refer to
+the following sections for type mapping in each direction.
+
+Druid type to Trino type mapping
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The connector maps Druid types to the corresponding Trino types according to the
+following table:
+
+.. list-table:: Druid type to Trino type mapping
+  :widths: 30, 30, 50
+  :header-rows: 1
+
+  * - Druid type
+    - Trino type
+    - Notes
+  * - ``STRING``
+    - ``VARCHAR``
+    -
+  * - ``FLOAT``
+    - ``REAL``
+    -
+  * - ``DOUBLE``
+    - ``DOUBLE``
+    -
+  * - ``LONG``
+    - ``BIGINT``
+    - Except for the special ``_time`` column
+  * - ``TIMESTAMP``
+    - ``TIMESTAMP``
+    - Only applicable to the special ``_time`` column
+
+Note that Druid does not have a real ``NULL`` value for any data type. By
+default, Druid treats ``NULL`` as the default value for a data type. For
+example, ``LONG`` would be ``0``, ``DOUBLE`` would be ``0.0``, ``STRING`` would
+be an empty string ``''``, and so forth.
+
+For more information, see `Druid data types
+<https://druid.apache.org/docs/latest/querying/sql-data-types.html>`_.
+
+No other data types are supported.
```

Comment:
Move this line to immediately below the table

Reply:
Done